v2/internal/controllers/dbformysql_flexibleserver_20231230_test.go

```diff
@@ -0,0 +1,245 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/kr/pretty"
+
+	mysql "github.com/Azure/azure-service-operator/v2/api/dbformysql/v1api20231230"
+	managedidentity "github.com/Azure/azure-service-operator/v2/api/managedidentity/v1api20181130"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1api20200601"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_DBForMySQL_FlexibleServer_20231230_CRUD(t *testing.T) {
+	t.Parallel()
```

Comment:
If we're pretty sure that this is correct but can't possibly record tests for it for some reason, we can always mark this `t.Skip()` and file an item to come back to it once we get to the root cause of why they're rejecting us...

Not ideal though.

Reply:
We got quota and I was able to record the tests.